lib/components/CustomSwitch.dart

```diff
@@ -119,14 +120,14 @@ class Track extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-        width: trackWidth,
-        height: trackHeight,
-        decoration: BoxDecoration(
-            borderRadius: $styles.borderStyle.trackBorderRadius,
-            color: widget.value ? activeColor : inactiveColor,
-            border: Border.all(
-                color: $styles.borderStyle.stroke,
-                width: $styles.borderStyle.width)));
```

Comment:
Noticed a lot of code duplication of this across the app, consolidated in $styles to reduce.